Examples/Python/src/TrackFitting.cpp

```diff
@@ -56,15 +56,37 @@ void addTrackFitting(Context& ctx) {
             .def(py::init<const Alg::Config&, Acts::Logging::Level>(),
                  py::arg("config"), py::arg("level"))
             .def_property_readonly("config", &Alg::config)
-            .def_static("makeTrackFitterFunction",
+            .def_static("makeKalmanFitterFunction",
                         py::overload_cast<
```

Comment:
So I used overloads to distinguish between directed and navigation with tracking geometry. Since this now doubles the number functions, do you think it would be better to check if the tracking geometry pointer is null, and use that to decide which navigator to set up?

Don't have a strong opinion on this, just a thought.

Reply:
Hmm with the current setup this is difficult on the C++ side, because these functions have different return types and thus cannot be overloaded.
Or do you have something else in mind?